update.sh

```diff
@@ -3,4 +3,5 @@ git pull
 yarn install --immutable
 git checkout -- yarn.lock
 yarn deploy
+cd /home/swk/plattform/ && node softwerkskammer/lib/migrateToSqlite/migrateAllFromMongo.js
```

Comment:
Why not do migrate the database before starting the server?

Reply:
platform is supposed to start thereafter by service demon. yarn deploy just "compiles"

Reply:
removed